Improve German translation

ChristianBeer opened the pull request:
Fixed some spelling errors, harmonized the translation of email to E-Mail and translated the remaining strings.

piRGoif commented:
Thanks for your contribution ! 

@larhip can you have a look ?

larhip commented on datamodels/2.x/combodo-backoffice-darkmoon-theme/dictionaries/de.dict.combodo-backoffice-darkmoon-theme.php:
```diff
@@ -22,5 +22,5 @@
-	'theme:darkmoon' => 'Dark moon~~',
-));
\ No newline at end of file
+	'theme:darkmoon' => 'Dunkler Mond',
```
From my point of view we should not translate the Name of the theme, and stick to the "original" English name. In French it is also not translated, maybe for the same reason @piRGoif ?

Hipska replied:
I agree with this as well, IMHO it shouldn't have been a translation item at all..

Molkobain replied:
Hello guys, I would tend to keep the themes name in english yes, it's like a feature or a color name (see phones color names for instance); and also because it would sound terrible in french to translate them 😅

larhip reviewed (commented):
Thanks a lot for your contribution!

There is just one issue regarding the translation of the theme name "dark moon", we should change.
The rest is fine from my point of view.

larhip commented:
Just checked it again. From content point of view everything is fine, many thanks!
Maybe you should just squash your commits into one but @piRGoif would know better, how it should be.

piRGoif commented:
> Just checked it again. From content point of view everything is fine, many thanks! 

Thanks for the verification Lars !

> Maybe you should just squash your commits into one but @piRGoif would know better, how it should be.

No need to squash, we'll do it when integrating the PR.
@ChristianBeer be sure though not to have any conflict with the base branch !

Speaking of, currently you are based on develop, which means we could integrate this for iTop 3.1.0. You might rebase on support/3.0 so we can integrate in iTop 3.0.2 ? (3.0.1 is close to release and I think we won't integrate PR for this version)
If you are ok, warning don't rebase only by changing the base branch in the PR. You must first rebase using git (rebase --onto), then force push, then change the PR base branch.

ChristianBeer commented:
I missed a string that started all this translation update. I added that too.

I'm fine with rebasing onto support/3.0. Does that mean it will automatically be merged into develop later? If not I could keep this PR open and updated for develop and create a new PR for 3.0.

Hipska commented:
@ChristianBeer they merge the support branches into develop from time to time.

Normally, if you rebase locally and force push to the same branch, you should be able to change the base branch in this PR by clicking Edit next to the title of this PR.

piRGoif commented:
Hello,

Yes we are merging on a regular basis from support/2.6 upwards (support/3.0, develop)

And yes no need to create a new PR, just do : 
1. local rebase (`git rebase --onto`)
2. force push
3. edit PR and change base branch

ChristianBeer commented:
I did the `rebase --onto` but somehow my commits got lost and I force pushed the head of `support/3.0` which closed this PR. Once I cherry-picked the dangling commits and pushed again I couldn't reopen this PR or change the base branch. So I created a new PR. Sorry for that.

piRGoif commented:
> So I created a new PR. Sorry for that.

Ok :)
Next time don't hesitate to ask us, we can do the rebase ourself if the "[allow edits from maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)" options is enabled.
